sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -458,10 +454,12 @@ def delete_configuration_setting(  # pylint:disable=delete-operation-wrong-retur
                 key=key,
                 label=label,
                 if_match=prep_if_match(etag, match_condition),
-                headers=custom_headers,
                 error_map=error_map,
+                **kwargs,
             )
-            return ConfigurationSetting._from_generated(key_value_deleted)  # type: ignore
+            if key_value_deleted:
```

Comment:
Is this the only case that config setting could be None?

e.g. what's the value if user wants to modify a non-existing config setting?

Reply:
* `delete` is the only operation that could get `None` from generated API. 
* Will get `None` when delete a non-existing config setting ([test](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/appconfiguration/azure-appconfiguration/tests/test_azure_appconfiguration_client.py#L197))